data/paper_title_abstract_example_file.json

```diff
@@ -0,0 +1 @@
+{"paper_sha": "", "title": "A Prototype-Based Few-Shot Named Entity Recognition", "abstract": "Few-shot Named Entity Recognition (NER) task focuses on identifying name entities on a small amount of supervised training data. The work based on prototype network shows strong adaptability on the few-shot NER task. We think that the core idea of these approaches is to learn how to aggregate the representation of token mappings in vector space around entity class. But, as far as we know, no such work has been investigated its effect. So, we propose the ClusLoss and the ProEuroLoss aiming to enhance the model's ability in terms of aggregating semantic information spatially, thus helping the model better distinguish entity types. Experimental results show that ProEuroLoss achieves state-of-the-art performance on the average F1 scores for both 1-shot and 5-shot NER tasks, while the ClusLoss has competitive performance on such tasks.", "journal_name": "Proceedings of the 8th International Conference on Computing and Artificial Intelligence"}
```

Comment:
I note that here we just have journal_name and not venue_name...

Reply:
Yes correcting this